Physics/Assets/Scripts/IP Transmission/Sender.cs

```diff
@@ -151,7 +151,41 @@ public void SendAsync(string data)
                 return;
             }
 
-            _sender.Connect(_remoteEndPoint);
+            int retryCount = 0;
+            while (retryCount < _maxRetries)
```

Comment:
retryCount is only incremented in the catch for SocketException. If a different type of exception is repeatedly encountered, will this loop indefinitely? Is that the intended behaviour?

Reply:
Was missing a return and continue to handle the exit. Other exceptions were supposed to exit immediately. Fixed in next commit.